### ISSUE TITLE
Error on install medical.patient.disease

### ISSUE THREAD

**Issue by lasley** [OCA/vertical-medical#61]
`medical.patient.disease` (most current version) errors out on initial install. I haven't T/S'd or anything yet, just dropping this here so I remember. Pulled from `Acsone/8.0-incubator-medical_disease-jne` 

```
ParseError: "ValidateError
Field(s) `arch` failed against a constraint: Invalid view definition

Error details:
Element '<xpath expr="//div[@name='buttons']">' cannot be located in parent view

Error context:
View `medical.patient.form (medical_disease)`
[view_id: 1702, xml_id: n/a, model: medical.patient, parent_id: 1679]" while parsing /usr/lib/python2.7/dist-packages/openerp/addons/medical_disease/views/medical_patient_view.xml:4, near
<record id="medical_patient_view_form" model="ir.ui.view">
            <field name="name">medical.patient.form (medical_disease)</field>
            <field name="model">medical.patient</field>
            <field name="inherit_id" ref="medical.view_medical_patient_form"/>
            <field name="arch" type="xml">
                <xpath expr="//div[@name='buttons']" position="inside">
                    <button name="%(medical_disease.medical_patient_disease_action)d" type="action" class="oe_stat_button" icon="fa-list" context="{'search_default_patient_id': active_id}">
                        <field name="count_disease_ids" widget="statinfo" string="Diseases"/>
                    </button>
                </xpath>
            </field>
        </record>
```

**Comment by JonathanNEMRY** [comment]
Hi @lasley !
very strange issue as we have this https://github.com/acsone/vertical-medical/blob/8.0-incubator_medical_disease-jne/medical/views/medical_patient_view.xml#L33 into the 8.0-incubator.

What are the other branches you use into your environement?
Is your 8.0-incubator is well rebase with the origin?

Unfortunatly I can't reproduce it from my side (I installed 8.0-incubator-medical_disease-jne from scratch and it works well)

Jonathan

**Comment by lasley** [comment]
Good question @JonathanNEMRY - I pulled directly from your branch with the pending PR, didn't do any rebases or anything. Everything was working great for me until I uninstalled everything (including the base medical module) then tried a reinstall. 

I didn't have much time to troubleshoot initially, and was honestly thinking it was my local install, but I just started with a fresh instance an it's giving the same issue.

What I don't understand is how the branch is passing the RunBot checks. Isn't that basically just an install on a blank instance?
